students/travis_nelson/lesson05/mailroom_part3.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+import sys
+
+
+donor_db = {"William Gates, III": [653772.32, 12.17],
+            "Jeff Bezos": [877.33, 3, 555, 132, 77, 1],
+            "Paul Allen": [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
+            "Brax Dingle": [2331, 32322.87, 5566.20, 3323.23, 76, 323, 3.87],
+            }
+
+
+def single_thank_you():
+    """Prints a letter to user-specified donor,
+    including recent and historical donation amounts"""
+    recipient = recipient_response()
+    donation = donation_response(recipient)
+    store_new_donation(recipient, donation)
+    total_donations = sum_donations(recipient)
+    print("Dearest {},\n"
+          "We are writing to formally thank you for your generous"
+          " donation of ${}.\n"
+          "To date, you have donated ${} to our honorable mission.\n"
+          "You are truly a valuable patron, and we thank you"
+          " for your service"
+          " (and money)".format(recipient, donation, total_donations))
+
+
+def recipient_response():
+    """Returns a recipient from the user...
+    If recipient was not previously in donor database, they are added"""
+    keep_asking = True
+    while keep_asking:
+        response = input("\nTo whom should we address this Thank You?\n"
+                         "(Type 'list' to view existing donors)\n\n").title()
+        try:
+            if response[0].isalpha():
+                if response.lower() == "list":
+                    print("\n")
+                    for i in donor_db:
+                        print("{}".format(i))
+                else:
+                    donor_exists = False
+                    for donor in donor_db:
+                        if donor == response:
+                            donor_exists = True
+                    if donor_exists:
+                        print("\nAn existing patron!\n")
+                        return response
+                    else:
+                        print("\nA new donor!\n")
+                        donor_db[response] = []
+                        return response
+            else:
+                raise TypeError
+        except TypeError:
+            print("\nType a name, plz\n")
```

Comment:
are you anticipating any other line to throw `TypeError` besides line 56?

if not, then you might as well print your string in line 56